modules/apps/digital-signature/digital-signature-lang/src/main/resources/content/Language.properties

```diff
@@ -1,11 +1,14 @@
 account's-base-uri=Account's Base URI
+add-document=Add Document
 add-recipient=Add Recipient
+added-documents=Added Documents
 api-account-id=API Account ID
 api-username=API User ID
 category.digital-signature=Digital Signature
 collect-digital-signature=Collect Digital Signature
 digital-signature=Digital Signature
 digital-signature-configuration-name=Digital Signature Configuration
+document-already-exist=Document already exist.
```

Comment:
Replace by **Document Already Exists** (Title)

Reply:
Hi Abelenda, this is an error message and not a title.